packages/chains/src/polygon/plugins.ts

```diff
@@ -79,6 +87,7 @@ const deployedPlugins: DeployedPlugins = {
       [underlying(assets, assetSymbols.MIMO)], // _rewardTokens
     ],
   },
+  // ArrakisERC4626_MIMO80-PAR20 BLP_0x82d7f08026e21c7713CfAd1071df7C8271B17Eae.json
```

Comment:
spaces in the file name are maybe ok, but I thought this is some kind of strategy code

Reply:
This is because the symbol value for the specific underlying is used. We use customised symbol files for some markets, so that the UI shows something prettier. THis is a bit sketchy, because if the symbol value changes at some point, it will break this file naming structure. But that's largely unlikely to happen for the plugins we have now.

See: https://github.com/Midas-Protocol/monorepo/blob/development/packages/types/src/enums.ts#L293